### PR TITLE
build fixed to download latest hazelcast and remote-controller

### PR DESCRIPTION
**Pull request by asimarslan**
Test Category is introduced for enterprise

### CODE REVIEW

**Comment by asimarslan** [comment]
verify

**Comment by asimarslan** [comment]
verify

**Comment by asimarslan** [comment]
verfiy

**Comment by asimarslan** [comment]
verify

**Comment by asimarslan** [comment]
verify

File: build.bat [hunk 1]
```diff
@@ -1,25 +1,48 @@
 @echo off
 setlocal EnableDelayedExpansion
 pushd %~dp0
-IF "%1"=="" (set COVERAGE="--no-coverage") ELSE (set COVERAGE=%1)
-set HAZELCAST_VERSION=3.7-SNAPSHOT
+
+IF -%1-==-- (set COVERAGE="--no-coverage") ELSE (set COVERAGE=%1)
+IF -%2-==-- (
+	set SERVER_TYPE=--oss
+	set EXLUDE_PARAM=/exclude:enterprise
```
Comment (cangencer) [inline build.bat]: minor typo, should be EXCLUDE

File: build.bat [hunk 2]
```diff
@@ -1,25 +1,48 @@
...
+) ELSE (
+	set SERVER_TYPE=%2
+	set EXLUDE_PARAM=
+)
+
+set HAZELCAST_VERSION=3.6.3
 set HAZELCAST_RC_VERSION=0.1-SNAPSHOT
 set HAZELCAST_HOME=%~dp0\server
```
Comment (cangencer) [inline build.bat]: this line is no longer used.

File: build.bat [hunk 3]
```diff
@@ -1,25 +1,48 @@
 @echo off
 setlocal EnableDelayedExpansion
 pushd %~dp0
-IF "%1"=="" (set COVERAGE="--no-coverage") ELSE (set COVERAGE=%1)
-set HAZELCAST_VERSION=3.7-SNAPSHOT
+
+IF -%1-==-- (set COVERAGE="--no-coverage") ELSE (set COVERAGE=%1)
```
Comment (cangencer) [inline build.bat]: I think this style of parameter passing will be problematic, if you specify for example build.bat --enterprise

**Comment by cangencer** [comment]
👍 (besides some minor typos/unused lines)

**Pull request merged**